contrib/python/pypodman/Makefile

```diff
@@ -1,4 +1,4 @@
-PYTHON ?= /usr/bin/python3
+PYTHON ?= $(shell command -v python3 2>/dev/null || command -v python)
```

Comment:
@wking One small request, and I'm not sure how to write the comment I want.  If you copy&paste that shell line into a shell you always get python.  It's the silenced fail of the "shell" command. Maybe it doesn't matter.

Reply:
> If you copy&paste that shell line into a shell you always get python. It's the silenced fail of the "shell" command.

Do you have `python3` in your `PATH`?  I do, and I see:

```console
$ command -v python3 2>/dev/null || command -v python
/usr/bin/python3
```

not `python`.  Exercising all possible cases:

```
$ for A in python3 does-not-exist-a
> do
>   for B in python does-no-exist-b
>   do
>     PYTHON=$(command -v "${A}" 2>/dev/null || command -v "${B}")
>     echo "${A} ${B} -> ${PYTHON}"
>   done
> done
python3 python -> /usr/bin/python3
python3 does-no-exist-b -> /usr/bin/python3
does-not-exist-a python -> /usr/bin/python
does-not-exist-a does-no-exist-b -> 
```

Are you expecting something else?

Reply:
What I attempted to explained badly was `echo $(shell command -v python3 2>/dev/null || command -v python)` always returns python.  I found it when I copied&pasted your new line without reading closely enough.  Like I said above, maybe it doesn't matter.

Reply:
> ... `echo $(shell command -v python3 2>/dev/null || command -v python)` always returns python

Ah, because [the `shell` function][1] is from Make.  It won't work if you call it directly from a POSIX shell.  To get more detail on the direct-in-the-shell invocation, you can drop the stderr redirect:

```console
$ echo $(shell command -v python3 || command -v python)
bash: shell: command not found
/usr/bin/python
```

If you want to paste into a POSIX shell, you need to pass the `shell` function's arguments without `shell` itself (like I did [here][2]).

[1]: https://www.gnu.org/software/make/manual/html_node/Shell-Function.html
[2]: https://github.com/containers/libpod/pull/818#discussion_r218498060